notebooks/idr0094-r.ipynb

```diff
@@ -0,0 +1,1441 @@
+       "28  Thonzonium bromide               1.28177015\n",
+       "29  Proflavine hemisulfate           2.10174716\n",
+       "30  Rociletinib (CO-1686, AVL-301)   0.07100162\n",
+       "⋮   ⋮                              ⋮           \n",
+       "303 SMI-4a                          0.83172680 \n",
+       "304 Endoxifen HCl                  11.98357416 \n",
+       "305 RG7112 (RO5045337)              0.05247987 \n",
+       "306 AZD3514                         1.32979979 \n",
+       "307 OSI-930                         1.07842382 \n",
+       "308 4-CMTB                          0.29877329 \n",
+       "309 PIK-75                          0.12723582 \n",
+       "310 QNZ (EVP4593)                   0.34679274 \n",
+       "311 AZD8055                         9.93249504 \n",
+       "312 WYE-354                         0.22095276 \n",
+       "313 Tocofersolan                    0.00344641 \n",
+       "314 Ku-0063794                      4.18122741 \n",
+       "315 GDC-0349                        0.04342338 \n",
+       "316 JNJ 1661010                     9.86323430 \n",
+       "317 CCT128930                       0.05393318 \n",
+       "318 AZD3463                         0.17872181 \n",
+       "319 OSI-027                         0.04038401 \n",
+       "320 Wnt-C59 (C59)                   3.12562789 \n",
+       "321 Torin 1                         0.18016227 \n",
+       "322 DBeQ                            0.01993226 \n",
+       "323 AZD7545                         0.04587587 \n",
+       "324 WAY-600                         1.36029155 \n",
+       "325 GSK2656157                      1.43811294 \n",
+       "326 Skepinone-L                    12.21639766 \n",
+       "327 MK-5108 (VX-689)                0.51440429 \n",
+       "328 GSK1059615                      0.44212409 \n",
+       "329 BMS 794833                      0.19518495 \n",
+       "330 SNX-2112                        0.28457287 \n",
+       "331 SC144                           0.11833250 \n",
+       "332 R547                            2.38173685 "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ic <- vector()\n",
+    "cn <- vector()\n",
+    "for(i in 1:nrow(compounds)) {\n",
+    "    c <- compounds[i,]\n",
+    "    if (c != \"\") {\n",
+    "      data <- filter_data(df, c)\n",
+    "      results <- calculate_IC50(data)\n",
+    "      ic <- c(ic, results$ic50)\n",
+    "      cn <- c(cn, c) \n",
+    "    }\n",
+    "}\n",
+    "result <- data.frame(cn, ic)\n",
+    "result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Export result as CSV\n",
+    "\n",
+    "Write the calculated IC50 result into a CSV file. The CSV file is first created in the ``home`` directory. The CSV file can then be downloaded to your local machine. To download it, click ``File > Open``, select the CSV file and open it, then click ``File > Download``."
```

Comment:
```suggestion
    "Write the calculated IC50 result into a CSV file. The CSV file is first created in the ``home`` directory. The CSV file can then be downloaded to your local machine. To download it, click ``File > Open``, navigate to the ``home`` directory (click onto the folder icon in top-left corner), select the CSV file and open it, then click ``File > Download``."
```

Reply:
For the results in the form of CSV, suggest we need to 

1. export better column header names, as commented on https://github.com/IDR/idr0094-ellinger-sarscov2/pull/1/files#r613280654
2. more importantly, there are no units for the ``ic`` column, which makes it less useful